### PR TITLE
lightning: add `enable-diagnose-logs` param

### PR DESCRIPTION
**Pull request by D3Hunter**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
- add `enable-diagnose-logs` param

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -31,6 +31,11 @@ file = "tidb-lightning.log"
 max-size = 128 # MB
 max-days = 28
 max-backups = 14
+
+# 是否开启 diagnose 日志，默认为 false，即只会输出跟导入有关的日志，不会输出依赖的其他组件的日志。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:35]:
```suggestion
# 是否开启 diagnose 日志。默认为 false，即只输出和导入有关的日志，不会输出依赖的其他组件的日志。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -31,6 +31,11 @@ file = "tidb-lightning.log"
...
+# 开启后会输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:36]:
```suggestion
# 设置为 true 开启后，会输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -31,6 +31,11 @@ file = "tidb-lightning.log"
 max-size = 128 # MB
 max-days = 28
 max-backups = 14
+
+# 是否开启 diagnose 日志。默认为 false，即只输出和导入有关的日志，不会输出依赖的其他组件的日志。
+# 设置为 true 开启后，会输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:36]: 是指既输出和导入相关的日志、又输出依赖的其他组件的日志吗？

Comment (D3Hunter) [reply tidb-lightning/tidb-lightning-configuration.md:36]: 是的

Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:36]:
```suggestion
# 设置为 true 开启后，既输出和导入相关的日志，也输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```

**Review by Frank945946 — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 4]
```diff
@@ -31,6 +31,11 @@ file = "tidb-lightning.log"
 max-size = 128 # MB
 max-days = 28
 max-backups = 14
+
+# 是否开启 diagnose 日志。默认为 false，即只输出和导入有关的日志，不会输出依赖的其他组件的日志。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:35]:
```suggestion
# 是否开启诊断日志。默认为 false，即只输出和导入有关的日志，不会输出依赖的其他组件的日志。
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -31,6 +31,11 @@ file = "tidb-lightning.log"
...
+# 设置为 true 开启后，既输出和导入相关的日志，也输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-configuration.md:36]:
```suggestion
# 设置为 true 后，既输出和导入相关的日志，也输出依赖的其他组件的日志，并开启 GRPC debug，可用于问题诊断。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**